### PR TITLE
Further preparation for catboost integration into library-bridge

### PR DESCRIPTION
**Pull request by rschu1ze**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

The individual commits contain more detailed descriptions.

The major change here is a new simple version field added to clickhouse-xdbc/library-bridges as discussed internally today.

### CODE REVIEW

File: programs/library-bridge/LibraryBridgeHandlers.cpp [hunk 1]
```diff
@@ -78,18 +80,41 @@ static void writeData(Block data, OutputFormatPtr format)
...
 
-void LibraryBridgeRequestHandler::handleRequest(HTTPServerRequest & request, HTTPServerResponse & response)
+void ExternalDictionaryLibraryBridgeRequestHandler::handleRequest(HTTPServerRequest & request, HTTPServerResponse & response)
 {
     LOG_TRACE(log, "Request URI: {}", request.getURI());
     HTMLForm params(getContext()->getSettingsRef(), request);
 
+    if (!params.has("version"))
+    {
+        processError(response, "No 'version' in request URL");
```
Comment (kssenii) [inline programs/library-bridge/LibraryBridgeHandlers.cpp:97]: may be regard version as 0 in this case? (e.g. version before versioning was added, we might deduce something from it)

File: programs/library-bridge/LibraryBridgeHandlers.cpp [hunk 2]
```diff
@@ -78,18 +80,41 @@ static void writeData(Block data, OutputFormatPtr format)
...
+        return;
+    }
+    else
+    {
+        String version_str = params.get("version");
+        size_t version;
+        auto [_, ec] = std::from_chars(version_str.data(), version_str.data() + version_str.size(), version);
```
Comment (kssenii) [inline programs/library-bridge/LibraryBridgeHandlers.cpp:104]: Better to use `tryParse<UInt64>(version_str, result)`

File: programs/library-bridge/LibraryBridgeHandlers.cpp (hunk 1 above)
Comment (rschu1ze) [reply programs/library-bridge/LibraryBridgeHandlers.cpp:97]: Yepp, done.

File: programs/library-bridge/LibraryBridgeHandlers.cpp (hunk 2 above)
Comment (rschu1ze) [reply programs/library-bridge/LibraryBridgeHandlers.cpp:104]: Done

**Review by kssenii — APPROVED** [review]

**Pull request merged**